### ISSUE TITLE
Isar Inspector 404

### ISSUE THREAD

**Issue by FritzMatthaeus** [isar-community/isar#72]
### Steps to Reproduce

I have opened the isar Inspector link but i am running into a 404. It has been working last week with the same configuration. 
I tried updating vom 3.1.6 to 3.1.7 but the 404 remained. 

### Code sample

```bash
I/flutter (24322): ╔════════════════════════════════════════════════════════════════╗
I/flutter (24322): ║                      ISAR CONNECT STARTED                      ║
I/flutter (24322): ╟────────────────────────────────────────────────────────────────╢
I/flutter (24322): ║              Open the link to connect to the Isar              ║
I/flutter (24322): ║             Inspector while this build is running.             ║
I/flutter (24322): ╟────────────────────────────────────────────────────────────────╢
I/flutter (24322): ║ https://isar-community.dev/inspector/3.1.7/#/62034/Fh-5mRvjCno ║
I/flutter (24322): ╚════════════════════════════════════════════════════════════════╝
```

opening the link will lead to the attached screenshot. There is nothing shown in the terminal logs.

### Details

 - Platform: MacOS 14.4.1,
 - Flutter version: 3.19.5
 - Isar version: 3.1.6 & 3.^7
 - Browser: Chrome 124.0.6367.119

---

 - [x] I searched for similar issues already
 - [x] I filled the details section with the exact device model and version
 - [ ] I am able to provide a reproducible example

![Bildschirmfoto 2024-05-06 um 10 42 14](https://github.com/isar-community/isar/assets/41049379/8581d999-51b8-4a5b-983b-924e1a8ddff8)

**Comment by dcarv01** [comment]
Same error here

**Comment by dcarv01** [comment]
Solution that work on my machine:

Change the url in src/isarConnect

from
final url = ' https://isar-community.dev/inspector/${Isar.version}/#/$port$path '

to
final url = ' https://inspect.isar-community.dev/${Isar.version}/#/$port$path ';

**Comment by FritzMatthaeus** [comment]
Thanks for taking care of this issue

**Comment by JCKodel** [comment]
> Solution that work on my machine:
> 
> Change the url in src/isarConnect
> 
> from final url = ' [https://isar-community.dev/inspector/${Isar.version}/#/$port$path](https://isar-community.dev/inspector/$%7BIsar.version%7D/#/$port$path) '
> 
> to final url = ' [https://inspect.isar-community.dev/${Isar.version}/#/$port$path](https://inspect.isar-community.dev/$%7BIsar.version%7D/#/$port$path) ';

This won't work on 4.0.3.

**Comment by mrclauss** [comment]
Yes, the github workflow was incorrect, so it did not deploy version 4.0.3
You may simply use inspector 4.0.2 instead

Am 7. Mai 2024 20:53:02 MESZ schrieb "J.C.Ködel" ***@***.***>:
>> Solution that work on my machine:
>> 
>> Change the url in src/isarConnect
>> 
>> from final url = '
>[https://isar-community.dev/inspector/${Isar.version}/#/$port$path](https://isar-community.dev/inspector/$%7BIsar.version%7D/#/$port$path)
>'
>> 
>> to final url = '
>[https://inspect.isar-community.dev/${Isar.version}/#/$port$path](https://inspect.isar-community.dev/$%7BIsar.version%7D/#/$port$path)
>';
>
>This won't work on 4.0.3.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/isar-community/isar/issues/72#issuecomment-2099096929
>You are receiving this because you modified the open/close state.
>
>Message ID: ***@***.***>

-- 
Martin Clauss
Meine Contactulater Card - All meine Kontaktinfo immer aktuell und vollständig:
https://contactulater.app/me/mc

**Comment by JCKodel** [comment]
> Yes, the github workflow was incorrect, so it did not deploy version 4.0.3 You may simply use inspector 4.0.2 instead

Also doesn't work =(

In debug window, that's the link I see:

`https://isar-community.dev/inspector/4.0.3/#/53942/0D_ibjOgdk8`

No one of those links work:

`https://isar-community.dev/inspector/4.0.3/#/53942/0D_ibjOgdk8`: 404 from isar-community.dev (black)
`https://isar-community.dev/inspector/4.0.2/#/53942/0D_ibjOgdk8`: 404 from isar-community.dev (black)
`https://inspect.isar-community.dev/4.0.3/#/53942/0D_ibjOgdk8`: 404 from GitHub (white)
`https://inspect.isar-community.dev/4.0.2/#/53942/0D_ibjOgdk8`: 404 from GitHub (white)

**Comment by vicenterusso** [comment]
Check the current published versions here

https://github.com/isar-community/inspector

Check something like this:
`https://inspect.isar-community.dev/4.0.0-dev.14/#/53942/0D_ibjOgdk8`

We will fix this ASAP.

**Comment by JCKodel** [comment]
@vicenterusso Thank you.
